src/main/resources/swagger/api-docs.yaml

```diff
@@ -329,6 +339,7 @@ definitions:
         description: Google's operation ID for the cluster
       status:
         type: string
+        enum: *CLUSTERSTATUS
```

Comment:
What syntax is this? C reference/pointer?

Reply:
It's a YAML anchor: https://swagger.io/docs/specification/2-0/enums/

Reply:
actually, i wonder if the merge key syntax will be less gross...

Reply:
According to the online swagger editor, no. Hmph.